tooling/reproducible/repro_common.sh

```diff
@@ -421,9 +430,6 @@ function cleanTemurinFiles() {
 
   echo "Cleaning Temurin build-scripts specific files and metadata from ${DIR}"
 
-  echo "Removing Temurin NOTICE file from $DIR"
-  rm "${DIR}"/NOTICE
```

Comment:
This should remain, cleanTemurinFiles() is called when comparing a Temurin build against a plain OpenJDK build, in which case Temurin specific metadata needs cleaning.

In this testcase scenario we are comparing Temurin vs Temurin